code/atoms/mobs.dm

```diff
@@ -83,21 +83,45 @@
 	dead_state = "player_dead"
 	rotatable = 1
 	inventory_type = /datum/inventory/player
+	var/is_super = FALSE
+	//speed at which the super animation runs, maximum 2 seconds to avoid epilepsy
+	var/super_animation_speed = 60
 
 /mob/living/inventory/player/die()
+	//removes any special statuses
+	icon_state = initial(icon_state)
```

Comment:
icon_state should get set to dead_state in /mob/living/die

Reply:
true!